### PR TITLE
Blackbox test parseSeconds

### PR DESCRIPTION
**Pull request by dianaamfr**

### CODE REVIEW

File: docs/02_assignment/02_assignment.md [hunk 1]
```diff
@@ -37,14 +43,39 @@
...
+Converts a time `String` in the format "\d+:[0-5]?\d:[0-5]?\d" (hours:minutes:seconds) to the equivalent time in seconds. Throws a `ParseException` if the format is not the expected.
 
 ### Category-Partition
-
-### Unit Tests
-
-### Outcome
+1. **Parameters/Input**: `strTime`, a `String` representing the time.
+2. **Characteristics of each parameter**: Considering the requirements, the parameter `strTime` must respect a specific time format "\d+:[0-5]?\d:[0-5]?\d" in order for the conversion to work correctly.
+3. **Constraints**: Given that the parameter is a `String`, it can assume infinite values, either respecting the requested format or not. Therefore, for the exceptional behavior we will consider only the cases where `strTime` is null, empty or has another format different from the accepted one.
```
Comment (Jumaruba) [inline docs/02_assignment/02_assignment.md:57]: Just needs to separate this point from the 4. Because they in the same line in markdown.

**Review by Jumaruba — APPROVED** [review]
lgtm!

File: docs/02_assignment/02_assignment.md (hunk 1 above)
Comment (dianaamfr) [reply docs/02_assignment/02_assignment.md:57]: Dei fix nisto, mudei para ParametrizedTest e acrescentei alguns casos

**Review by Jumaruba — APPROVED** [review]
lgtm again!

**Pull request merged**